src/EventManager.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Castor;
+
+use Symfony\Component\EventDispatcher\EventDispatcher;
+
+class EventManager
```

Comment:
Original message from @lyrixx 
> Why do we need this class (EventManager) ? let's use Symfony EventDispatcher directly?

As @tigitz say, this will allow us to additionally log event registrations and triggers

Reply:
Okay, why not. But let's use decoration instead:
* Name the class EventDispatcher (Manager is usually a bad name)
* Implements EventDispatcherInterface